plugins/modules/perlmod_install_info.py

```diff
@@ -0,0 +1,410 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2023, Jonathan Kamens <jik@kamens.us>
```

Comment:
```suggestion
# Copyright (c) 2024, Jonathan Kamens <jik@kamens.us>
```

Reply:
I would just remove the year to avoid having to update this, dates present in the commit history.  

Also we've started attributing to `The Ansible Project'  in core, while updating the 'authors' field in docs.

Reply:
1. Copyright notices that do not contain the year are not valid under international copyright law.
2. Is it your intention to assert that authors of modules contributed to Ansible are required to relinquish their copyright to The Ansible Project? If so, that's news to me, and somewhat concerning; is this policy spelled out somewhere? If this is not the case, then it's not appropriate to specify "The Ansible Project" rather than the actual copyright-holder.

Reply:
No, IANAL, it was legal advice we got in core and followed, feel free to disregard, specially since this is a community  managed project. My intent was to make future updates less painful.